### DISCUSSION TITLE
Not Configure 2FA

### DISCUSSION THREAD

**Discussion by lexionq** [community/community#140910] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

 I tried enable 2FA with Microsoft Authenticator. But Github says ***"Two-factor code verification failed. Please try again."*** But the verification code is true.

**Comment by VeerammalV** [comment]
Ensure that the time on your device is synced correctly. Authentication apps like Microsoft Authenticator rely on accurate time settings, and if the device’s time is off, even by a few seconds, the verification code may fail. You can try enabling automatic time sync on your phone or manually adjust the time. If the issue persists, try removing and re-adding your GitHub account in the Microsoft Authenticator app, then attempt the 2FA setup again.

**Reply by LuisaConsiglieri** [reply]
I have the same problem. How do I sync the time between my laptop and the Chrome authenticator? Thank you in advance.

**Comment by lexionq** [comment, marked as answer]
I try enable 2FA with other device. And its succesfully verified.

**Comment by TomEdwardsEnscape** [comment]
I was having this problem and eventually noticed that there is a "Verify the code from the app" field beneath the QR code that I had (repeatedly) scanned.

This doesn't mean "verify that the code works", it means "you must type the code from here to complete registration of your device, otherwise every code it produces will be rejected at login time".

If you haven't entered a code and then clicked the button then your device is not yet registered. This isn't obvious at all.
